plugins/include/amxmodx.inc

```diff
@@ -1828,7 +1828,7 @@ native set_user_flags(index, flags = -1, id = 0);
  * @param index     Client index, 0 to set flags of server
  * @param id        Flag set id, ranging from 0 to 31
  *
- * @noreturn
+ * @return          Bitflag sum of client's admin flags, 0 if client is not connected
```

Comment:
You can remove `, 0 if client is not connected` ; because 0 is only returned when there is an error (not connected), this return value can't be used.

Reply:
You should not explicit return values which can't be used.

If you look at `get_user_weapons` source code, it errors on an invalid index, but it returns 0 if you are not in-game. The return value can be used.